### PR TITLE
ref(controller): SLUG_RUNNER_IMAGE_PULL_POLICY got removed and IMAGE_PULL_POLICY is the new thing to use

### PR DESCRIPTION
**Pull request by helgi**
https://github.com/deis/controller/pull/1053 for rationale

### CODE REVIEW

File: workflow-dev/tpl/generate_params.toml [hunk 1]
```diff
@@ -45,6 +45,17 @@ logger_redis_location = "on-cluster"
...
+# Set the default (global) way of how Application (your own) images are
+# pulled from within the Controller.
+# This can be configured per Application as well in the Controller.
+#
+# This affects pull apps and git push (slugrunner images) apps
+#
+# Values values are:
+# - Always
+# - IfNotPresent
+image_pull_policy = "Always"
```
Comment (kmala) [inline workflow-dev/tpl/generate_params.toml]: it would be better  to have it as part of the controller as it is related to only it

Comment (helgi) [reply workflow-dev/tpl/generate_params.toml]: Thought about that but we already have a `pullPolicy` there which is only related to the controller image itself, which seems confusing at best if we have 2 very similarly named things there

Comment (kmala) [reply workflow-dev/tpl/generate_params.toml]: can we change to app_pull_policy then?

Comment (helgi) [reply workflow-dev/tpl/generate_params.toml]: Check now

**Pull request merged**